### PR TITLE
Implemented endpoint to fetch playlists from your channel

### PR DESCRIPTION
**Pull request by devos50**

### CODE REVIEW

**Comment by devos50** [comment]
@lfdversluis ready for your review.

File: doc/Tribler REST API.md [hunk 1]
```diff
@@ -179,6 +180,26 @@ Add a RSS feed to your channel. Returns error 409 (Conflict) if the supplied RSS
 
 Delete a RSS feed from your channel. Returns error 404 if the RSS feed that is being removed does not exist. Note that the rss feed url should be URL-encoded.
 
+## `GET /mychannel/playlists`
+
+Return the playlists in your channel. Returns error 404 if you have not created a channel.
```
Comment (lfdversluis) [inline doc/Tribler REST API.md]: Returns

**Comment by lfdversluis** [comment]
One type, rest looks good.

**Comment by devos50** [comment]
@lfdversluis thanks! @whirm can you review? I will fix both your and Laurens' feedback after that.

**Comment by whirm** [comment]
Looks fine to me

**Comment by devos50** [comment]
retest this please

**Comment by devos50** [comment]
retest this please

**Comment by devos50** [comment]
retest this please

**Comment by devos50** [comment]
retest this please

**Comment by devos50** [comment]
@whirm ready for merge

**Pull request merged**